crypto/keccak_ziren.go

```diff
@@ -118,5 +118,5 @@ func Keccak256(data ...[]byte) []byte {
 
 // Keccak256Hash calculates and returns the Keccak256 hash as a Hash using the Ziren zkvm_runtime implementation.
 func Keccak256Hash(data ...[]byte) common.Hash {
-	return common.Hash(Keccak256(data...))
```

Comment:
I think it makes sense, we can't directly convert the byte slice to byte array by `common.Hash(slice)`

```go
func TestXX(t *testing.T) {
	xx := []byte{0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0x7, 0x8}
	fmt.Println(common.Hash(xx))
}
```

Compiler reports something 

```
panic: runtime error: cannot convert slice with length 8 to array or pointer to array with length 32

```

Reply:
But the returned slice has length 32, and it can be converted.